resources/contribute-documentation/index.md

```diff
@@ -153,20 +146,59 @@ Use the following words and phrases as described:
 
 ## Frontmatter Fields
 
-***Note: The details of Realm's frontmatter specification are not fully documented. Update this with a link when Realm exits closed beta.***
-
 Frontmatter for Markdown files can include details such as the following:
 
 ```yaml
 ---
 metadata:
-  indexPage: true # Add this if you want the page to contain an auto-generated list of its child pages.
+    indexPage: true # Use if you want to use the {% child-pages /%} macro to display a list of page children
+seo:
+    description: Transact with confidence using the XRP Ledger's suite of compliance features for following government regulations and security practices.
+    title: Compliance Features
```

Comment:
Suggest noting that the title field here is optional, and moving the note about when to use this field from line 171 below.